### DISCUSSION TITLE
Remove Featured Section

### DISCUSSION THREAD

**Discussion by roime81** [ckan/ckan#7793] [Q&A]
Hi, 

I want to remove Featured Section and Stats section in the frontpage. How can I doing it ?. Is that can be done by changing CSS ?

**Comment by Pooya-Oladazimi** [comment]
You can create a new ckan extension/plugin to overwrite the related pages with the new content. 
for instance, you can overwrite this snippet https://github.com/ckan/ckan/blob/master/ckan/templates/home/snippets/promoted.html 
or any other template file that is needed in your plugin:
docs: https://docs.ckan.org/en/2.9/contributing/frontend/templating.html?highlight=template#ckan-extends
